File: SkaiLab.Invoice/ClientApp/src/app/models/tax.ts
export class Tax{
    id:number;
    name:string;
    organisationId:string;
    totalRate:number;
    components:TaxComponent[];
    constructor(){
        this.id=0;
        this.name="";
        this.organisationId="";
        this.totalRate=0;
        this.components=[];
    }
}
export class TaxComponent{
    id:number;
    name:string;
    rate:number;
    constructor(){
        this.id=0;
        this.name="";
        this.rate=0;
    }
}

File: SkaiLab.Invoice.Models/Tax.cs
using System.Collections.Generic;
namespace SkaiLab.Invoice.Models
{
    public class Tax
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string OrganisationId { get; set; }
        public decimal TotalRate { get; set; }
        public List<TaxComponent> Components { get; set; }

    }
    public class TaxComponent
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public decimal Rate { get; set; }
    }
}
